scripts/generate-and-apply-harbor-yaml.sh

```diff
@@ -109,7 +110,8 @@ tanzu package install harbor \
 
 # Patch (via overlay) the httpproxy (contour) timeout for pulling down large images.  Required for TBS which has large builder images
 kubectl create secret generic harbor-timeout-increase-overlay -n tanzu-kapp -o yaml --dry-run=client --from-file=tkg-extensions-mods-examples/registry/harbor/overlay-timeout-increase.yaml | kubectl apply -f -
-kubectl annotate PackageInstall harbor -n tanzu-kapp ext.packaging.carvel.dev/ytt-paths-from-secret-name.0=harbor-timeout-increase-overlay
+kubectl annotate PackageInstall harbor -n tanzu-kapp ext.packaging.carvel.dev/ytt-paths-from-secret-name.0=harbor-timeout-increase-overlay --overwrite
```

Comment:
Wasn't working for you without the "--overwrite"?

Reply:
It did work for me, but only the first time I ran the script.  I wanted to be able to run the script multiple times, which required the overwrite.